.github/workflows/nighly_fuzzing.yml

```diff
@@ -0,0 +1,56 @@
+name: keth-NIGHTLY-FUZZING
+
+on:
+  schedule:
+    - cron: 20 0 * * *
+  workflow_dispatch: {}
+
+permissions: read-all
+
+jobs:
+  tests-unit:
+    runs-on: ubuntu-latest-16-cores
+    timeout-minutes: 360
+    env:
+      PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION: python
+    steps:
+      - uses: actions/checkout@v4
+      - uses: astral-sh/setup-uv@v2
+        with:
+          enable-cache: true
+          cache-dependency-glob: uv.lock
+      - uses: actions/setup-python@v5
+        with:
+          python-version-file: ./cairo/.python-version
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
+      - name: Run tests
+        id: tests
+        continue-on-error: true # To send a notification if the tests fail
+        run: |
+          echo "HYPOTHESIS_PROFILE=nightly" >> .env
+          cd cairo
+          uv run compile
+          forge build
+          uv run pytest "tests" -m "not NoCI" -n logical --seed 42 --junitxml=junit.xml -o junit_family=legacy
```

Comment:
I don't know if we want/need to have a seed 
Does it influence hypothesis input generation ?

Reply:
I think it only determines the seed for the random number generators, not the test themselves. you're right that it's probably best without.